pep-0684.rst

```diff
@@ -1,13 +1,13 @@
 PEP: 684
 Title: A Per-Interpreter GIL
 Author: Eric Snow <ericsnowcurrently@gmail.com>
-Discussions-To: python-dev@python.org
+Discussions-To: `python-dev@python.org <https://mail.python.org/archives/list/python-dev@python.org/thread/CF7B7FMACFYDAHU6NPBEVEY6TOSGICXU/>`__
```

Comment:
```suggestion
Discussions-To: https://mail.python.org/archives/list/python-dev@python.org/thread/CF7B7FMACFYDAHU6NPBEVEY6TOSGICXU/
```

The link text is generated automatically. I already have updates to the linters to catch this at commit time, they just haven't been merged yet.

Reply:
People should post to that thread on python-dev, not the mailing list archive.  Hence the link.

Reply:
Do you mean it automatically make the link text "python-dev@python.org"?